plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -79,4 +81,17 @@ public void testShowCreateTable()
                         format("   location = '.*/" + schemaName + "/region'\n") +
                         "\\)");
     }
+
+    @Test
+    public void testHiddenPathColumn()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "hidden_path_column", "(a int, b VARCHAR)", ImmutableList.of("(1, 'a')"))) {
+            String filePath1 = (String) computeScalar(format("SELECT file_path FROM \"%s$files\"", table.getName()));
```

Comment:
Let's rename to just `filePath`

Reply:
done changes @ebyhr

Reply:
@ebyhr changed to  `filePath`

Reply:
I meant the variable name, `String filePath1` → `String filePath`